x-pack/plugins/security_solution/server/lib/detection_engine/signals/threat_mapping/build_threat_mapping_filter.ts

```diff
@@ -108,7 +117,7 @@ export const createAndOrClauses = ({
   threatMapping,
   threatListItem,
   entryKey,
-}: CreateAndOrClausesOptions): BooleanFilter => {
+}: CreateAndOrClausesOptions): unknown[] => {
```

Comment:
why it becomes `unknown`? is there according type for `should` clause? Probably `QueryDslQueryContainer`?